autoload/tagbar.vim

```diff
@@ -1952,7 +1952,7 @@ function! s:PrintKinds(typeinfo, fileinfo) abort
             for tag in curtags
                 call s:PrintTag(tag, 0, output, a:fileinfo, a:typeinfo)
 
-                if !g:tagbar_compact
+                if g:tagbar_compact != 1
```

Comment:
Do we need to test here for the case where the variable is not set at all? Is being set to *something* a guarantee at this point due to the way we load preferences or might this not exist at all?

Reply:
No tests needed. All the variables get initialized to their defaults in `tagbar/plugin/tagbar.vim`.